searchlib/src/vespa/searchlib/fef/termmatchdatamerger.cpp

```diff
@@ -41,31 +41,52 @@ TermMatchDataMerger::merge(uint32_t docid,
 {
     _scratch.clear();
     bool wasMatch = false;
+    bool needs_normal_features = out.needs_normal_features();
```

Comment:
Please extend searchlib/src/tests/fef/termmatchdatamerger/termmatchdatamerger_test.cpp with new test cases for the new functionality that is added.

Reply:
term match data merger test has been extended.